### DISCUSSION TITLE
Gamerule GUI improvement

### DISCUSSION THREAD

**Discussion by Memory-Yzf** [Zrips/CMI#8116] [Enhancement]
Some mods add new game rules

Using a hybrid server core can fill up the entire GUI causing the original buttons to be overwritten

![1](https://github.com/Zrips/CMI/assets/102323087/448b6f40-7a2f-4810-9305-7f92a6a17246)

**Comment by CloudeLecaw** [comment]
Hybride servers are normaly not supported..

But yes..if Mohjang starts adding more gamerules then the GUI can show at least. then it would be time for multiple pages :D

As long as Vanilla hasnt added more gamerules then 9x5 (so 45) can show as long i dont see Zrips adding multiple
versions of the GUI for each exsting server hybrid ;)
